src/main/java/com/joyent/manta/monitor/Application.java

```diff
@@ -49,21 +48,23 @@ public static void main(final String[] args) throws InterruptedException {
             System.exit(1);
         }
 
+        int jettyServerPort = validateJettyServerPort(System.getenv("JETTY_SERVER_PORT"));
         final URI configUri = Objects.requireNonNull(parseConfigFileURI(args[0]));
         final MantaMonitorModule module = new MantaMonitorModule(UNCAUGHT_EXCEPTION_HANDLER,
                 UNCAUGHT_EXCEPTION_HANDLER.getReporter(), configUri);
         final MantaMonitorServletModule mantaMonitorServletModule = new MantaMonitorServletModule();
         final Injector injector = Guice.createInjector(module);
         final Configuration configuration = injector.getInstance(Configuration.class);
-        final Injector jettyServerBuilderInjector = injector.createChildInjector(new JettyServerBuilderModule(injector), mantaMonitorServletModule);
-
+        final Injector jettyServerBuilderInjector = injector.createChildInjector(new JettyServerBuilderModule(jettyServerPort), mantaMonitorServletModule);
```

Comment:
Separate the instantiation of `JettyServerBuilderModule`, so that the stacktrace is easier to follow when there is an exception.